doc/content/reference/root-certificates/_index.md

```diff
@@ -7,6 +7,12 @@ This section contains links to common root SSL certificates used in {{% tts%}},
 
 <!--more-->
 
+## Let's Encrypt
+
+Most {{% tts %}} deployments use the Let's Encrypt ISRG Root X1 Trust. If using Let's Encrypt to secure your domain, you may download the ISRG Root X1 Trust file [here](https://letsencrypt.org/certs/isrgrootx1.pem).
```

Comment:
Let's say that "Many" deployments use it (TTI Cloud Hosted actually still uses a chain with `DST Root CA X3`)

I really think the order of recommendation should be:

1. Full list of commonly trusted CAs
2. Minimal list of frequently used CAs in The Things Stack deployments
3. Single CA